### DISCUSSION TITLE
Gestione del finale di stagione

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#49] [General]
Fossa si è giustamente lamentato del fatto che ad Agosto non escono episodi di Joypad e quindi occorre gestire il finale di stagione.

**Comment by pdonadeo** [comment]
Ho sistemato anche questo piccolo difetto. Per vederne gli effetti toccherà aspettare il prossimo Agosto. L'effetto comunque sarà questo:

![screenshot_2022-09-11T11 35 26](https://user-images.githubusercontent.com/242049/189521503-515c046f-79e2-426e-b7cf-32b98ce9473a.png)

**Reply by agofa** [reply]
Bene. Sistemato anche Fossa 😍
